### ISSUE TITLE
Initial state of BTenabled incorrect in iOS 9.1

### ISSUE THREAD

**Issue by ryami333** [255kb/cordova-plugin-bluetooth-status#2]
Using in a cordova/ionic project, I boot the app on an ipad-3 running iOS 9, and get the following, despite bluetooth being enabled and verified working:

```
cordova.plugins.BluetoothStatus.BTenabled;
// false
```

Then, I turn the bluetooth off and on again, within the device's settings panel:

```
cordova.plugins.BluetoothStatus.BTenabled;
// true
```

It seems the method only returns the correct state after triggering an update by manually changing the device's bluetooth settings. Invoking `cordova.plugins.BluetoothStatus.initPlugin` does not affect the initial state.

**Comment by mrlowe** [comment]
@ryami333, I'm seeing this exact issue in a Meteor app. If you [look at the code](https://github.com/255kb/cordova-plugin-bluetooth-status/blob/master/src/ios/BluetoothStatus.m#L19), `cordova.plugins.BluetoothStatus.initPlugin` actually doesn't do anything on the iOS side, so it looks like there might be a problem with pluginInitialize not getting called on the native object.

**Comment by 255kb** [comment]
Hi, 

it seems that there is definitely a problem with iOS 9+. it was working in previous versions. Also, initial state of the bluetooth adapter should be populated by [line 47](https://github.com/255kb/cordova-plugin-bluetooth-status/blob/master/src/ios/BluetoothStatus.m#L47) `[self centralManagerDidUpdateState: self.bluetoothManager];`. But maybe it is too early and javascript is not ready to receive the information.

I will have a look asap and test something else.

**Comment by beaur** [comment]
Would love to know when this issue is resolved so that I can use this plugin in my app

**Comment by 255kb** [comment]
I have published a new version 1.0.4 where the bug should be fixed. I have tested it on various iOS9+ devices. Please let me know if it is working for you. 

The `initPlugin()` method should definitely be called now (on both Android and iOS) to ensure that the initial state of the bluetooth adapter is correctly populated. You can call this method wherever you want in your javascript as it is waiting for the `deviceready` event anyway.
